applauncher/src/main/java/de/interseroh/tmb/applauncher/client/ApplauncherWebApp.java

```diff
@@ -52,27 +60,28 @@
 			.create(ApplauncherWebAppGinjector.class);
 
 	private AppConfigurationClient appConfigurationClient;
+	private String applauncherUrl;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "applauncherUrl" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)